### PR TITLE
You can now shoot narsie

### PR DESCRIPTION
**Pull request by MadmanMartian**
image

:cl:
 * rscadd: you can now shoot narsie, requires a null rod fired from a railgun above a gigawatt of charge

### CODE REVIEW

**Comment by bathosbathtime** [comment]
Yes.  _YES._

**Comment by spoonylove** [comment]
its gotta make the BFG sound

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -429,6 +429,13 @@
 	if(istype(atarget, /mob/living) && damage == 200)
 		var/mob/living/M = atarget
 		M.gib()
+	else if(istype(atarget, /obj/machinery/singularity/narsie) && blessed && damage == 200) //MINE IS THE ROD THAT SHALL PIERCE THE HEAVENS
+		var/obj/machinery/singularity/narsie/N = atarget
+		if(!N.wounded)
+			N.visible_message("<span class = 'danger'>\The [src] strikes \the [N], wounding them. This god can bleed!</span>", range = 20)
```
Comment (Kurfursten) [inline code/modules/projectiles/projectile/bullets.dm:435]: I'm a bit disappointed this is all that happens for a setup that is probably one in a million to happen.

File: code/modules/projectiles/guns/projectile/constructable/railgun.dm [hunk 2]
```diff
@@ -204,7 +211,7 @@
 		else
 			to_chat(user, "<span class='warning'>\The [C] is not charged.</span>")
 	if(rod_loaded)
-		to_chat(user, "<span class='info'>There is \a metal rod loaded into the barrel.</span>")
+		to_chat(user, "<span class='info'>There is a [rod_loaded == "Holy"? "null rod":"metal rod"] loaded into the barrel.</span>")
```
Comment (jknpj) [inline code/modules/projectiles/guns/projectile/constructable/railgun.dm:214]: get the null rod's name

Comment (jknpj) [reply code/modules/projectiles/guns/projectile/constructable/railgun.dm:214]: actually they get deleted but this still breaks my immulshion

File: code/modules/projectiles/projectile/bullets.dm (hunk 1 above)
Comment (MadmanMartian) [reply code/modules/projectiles/projectile/bullets.dm:435]: Well, depends on how many times narsie is shot overall over multiple rounds. Deity didn't want narsie being absolutely destroyed as he's a 'constant' between rounds.

We'll see for the future.

Comment (Kurfursten) [reply code/modules/projectiles/projectile/bullets.dm:435]: You don't have to destroy him, you could banish him back or weaken the veil causing all the summoned monsters to go away.

Comment (DeityLink) [reply code/modules/projectiles/projectile/bullets.dm:435]: That'd be crew victory. You don't banish an elder god with a railgun, it's less about reversing the tide and more about bragging rights for having managed to wound Nar-Sie. The round is over anyway at that point.

**Comment by Durgen** [comment]
How big is the hitbox?

**Comment by MadmanMartian** [comment]
1 tile, are you a bad enough dude?

File: code/modules/projectiles/projectile/bullets.dm (hunk 1 above)
Comment (brndd) [reply code/modules/projectiles/projectile/bullets.dm:435]: >You don't banish an elder god with a railgun

b-but to defeat the Cyberdemon...

File: code/modules/projectiles/guns/projectile/constructable/railgun.dm [hunk 3]
```diff
@@ -177,9 +180,13 @@
...
-		var/obj/item/stack/rods/R = W
-		rod_loaded = 1
-		R.use(1)
+		if(istype(W, /obj/item/stack/rods))
+			var/obj/item/stack/rods/R = W
+			R.use(1)
+			rod_loaded = 1
+		else if(istype(W, /obj/item/weapon/nullrod))
+			qdel(W)
+			rod_loaded = "Holy"
```
Comment (PJB3005) [inline code/modules/projectiles/guns/projectile/constructable/railgun.dm:189]: Excuse me but what the fuck is this? How did this shit get merged?

Comment (brndd) [reply code/modules/projectiles/guns/projectile/constructable/railgun.dm:189]: \>party member of previous game returns as an antagonist in the sequel

**Pull request merged**